layouts/partials/nav.html

```diff
@@ -64,10 +64,55 @@
             {{ end }}
 
             <li class="ms-2">
-                <button class="btn donate-btn btn-outline-primary text-uppercase px-4 rounded-0"
-                        type="button" style="font-size: 0.9rem;">
+                <button class="btn donate-btn btn-outline-primary text-uppercase px-4 rounded-0" type="button" data-bs-toggle="modal" data-bs-target="#donateModal" style="font-size: 0.9rem;">
                     Donate
-                </button>
+                  </button>
+                  
+                  <!-- Donate Modal -->
```

Comment:
## CRITICAL

Can't see anything on mobile phone.

On smaller screen sizes, modal may not work well anymore, u have to consider alternative ways to display the info.

Reply:
What do you suggest?

Reply:
> What do you suggest?

@lynngoh7 maybe this ig

![Screenshot_20240725_083445_Chrome.jpg](https://github.com/user-attachments/assets/64a25013-1739-4080-99c7-c4fb76c4f250)

Reply:
@lynngoh7 maybe this ig
![Screenshot_20240725_083445_Chrome.jpg](https://github.com/user-attachments/assets/dbdc337e-a030-47df-9331-a202288eed93)